encryption/ntru/ntru-kem/param/ntruhps2048677.yaml

```diff
@@ -0,0 +1,14 @@
+name: NTRU-HPS-2048-677
+security level:
+  quantum: 144
+  nist category: 1
+  comment: >-
+    Assuming a non-local model of computation.
+    (With a local model: 200 bits quantum security, category 3)
+sizes:
```

Comment:
In the paper the numbers are one byte bigger (Sec. 3.2).

Reply:
They contradict themselves. I got the numbers from § 1.6. Supercop confirms this (https://bench.cr.yp.to/results-kem.html, 5th result when searching for `ntruhps2048677`)